src/elements/selection-action-panel/selection-action-panel.scss

```diff
@@ -0,0 +1,75 @@
+@use '../core/styles' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  --sbb-selection-action-panel-background: var(--sbb-color-white);
+  --sbb-selection-action-panel-border-color: var(--sbb-color-cloud);
+
+  // Variable used to override background color of selection panels.
+  --sbb-selection-panel-inner-background: transparent;
+
+  // Variable used to override border width of selection panels.
+  --sbb-selection-panel-inner-border-width: 0px;
+  --sbb-selection-action-panel-border-width: var(--sbb-border-width-1x);
+  --sbb-selection-action-panel-border-radius: var(--sbb-border-radius-4x);
+
+  // As the selection panel has always a white/milk background, we have to fix the focus outline color
+  // to default color for cases where the selection panel is used in a negative context.
+  --sbb-focus-outline-color: var(--sbb-focus-outline-color-default);
+
+  display: contents;
+}
+
+:host([color='milk']) {
+  --sbb-selection-action-panel-background: var(--sbb-color-milk);
+}
+
+:host(:where(:state(checked), [state--checked]):not(:where(:state(disabled), [state--disabled]))) {
+  --sbb-selection-action-panel-border-color: var(--sbb-color-charcoal);
+  --sbb-selection-action-panel-border-width: var(--sbb-border-width-2x);
+}
+
+:host([borderless]:not(:where(:state(checked), [state--checked]))) {
+  --sbb-selection-action-panel-border-color: transparent;
+}
+
+:host(:is(:state(with-expansion-panel), [state--with-expansion-panel])) {
+  --sbb-selection-action-panel-background: transparent;
+  --sbb-selection-action-panel-border-color: transparent;
+}
+
+::slotted(:is(sbb-checkbox-panel, sbb-radio-button-panel)) {
+  flex: auto;
+}
+
+::slotted(:is(sbb-secondary-button, [data-action])) {
+  align-self: center;
+  padding-inline: var(--sbb-spacing-responsive-xxs);
```

Comment:
Padding has the effect, that there is an invisible click area before and after the button.
Would margin-inline also work?